servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.h

```diff
@@ -76,11 +79,20 @@ class RenderSceneDataRD {
 	float time;
 	float time_step;
 
+	virtual Transform3D get_cam_transform() const override;
+	virtual Projection get_cam_projection() const override;
+
+	virtual uint32_t get_view_count() const override;
+	virtual Vector3 get_view_eye_offset(uint32_t p_view) const override;
+	virtual Projection get_view_projection(uint32_t p_view) const override;
```

Comment:
We should also expose the inv proj & view matrix. Right now there is no easy way for people to calculate them since we don't expose affine inverse function to gdscript.

Reply:
Is there a reason we're not exposing affine inverse? We're not applying the inverse until we load the data into the UBO so any such access method would just be performing the affine inverse function on these matrices.
I guess we could add these functions for convenience but they'll have a hidden cost.

Reply:
Btw, on this subject, I want to expose much more info so we can do some of this in follow up PRs. Makes it easier to review things and not get bogged down in what we should and shouldn't include in the main PR.

Reply:
I haven't had trouble inverting the matrices. I use Transform3D.affine_inverse() and Projection.inverse()

Reply:
Ultimately, I think we need to expose a way for the user to request the built in uniform set/data buffers. I would merge the PR for now and then expose exactly what we want once we have feedback from users on what is missing